packages/babel-traverse/src/path/index.ts

```diff
@@ -109,7 +109,7 @@ class NodePath<T extends t.Node = t.Node> {
   }
 
   getScope(scope: Scope): Scope {
-    return this.isScope() ? new Scope(this) : scope;
+    return this.isScope() ? new Scope(this as any) : scope;
```

Comment:
Can we add a comment here that explains why `as any`, linking to this PR? Also `// TODO: Remove this when TS is fixed`, so that we don't leave it forever (every few months I check our `TODO:` comments to see what can be easily removed :grin:)